<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
	background-color: qlineargradient(spread:pad, x1:0.063, y1:0.068, x2:0.744318, y2:0.83, stop:0 rgba(0, 255, 127, 255), stop:1 rgba(90, 90, 90, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>611</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;Arial&quot;;
color: black;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;REGISTER SECONDARIES &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;NODES&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblName">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Arial&quot;;
color: black;</string>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblHost">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Arial&quot;;
color: black;</string>
    </property>
    <property name="text">
     <string>Host:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPort">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Arial&quot;;
color: black;</string>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDatabase">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Arial&quot;;
color: black;</string>
    </property>
    <property name="text">
     <string>Database:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblUser">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Arial&quot;;
color: black;</string>
    </property>
    <property name="text">
     <string>User:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPassword">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Arial&quot;;
color: black;</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Node1</string>
    </property>
    <property name="placeholderText">
     <string>Node name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputHost">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>localhost</string>
    </property>
    <property name="placeholderText">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPort">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>5432</string>
    </property>
    <property name="placeholderText">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputDatabase">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>CTEC</string>
    </property>
    <property name="placeholderText">
     <string>Database name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputUser">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>postgres</string>
    </property>
    <property name="placeholderText">
     <string>User name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPassword">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>12345</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>User password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInsert">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
font: 12pt &quot;Arial&quot;;
background-color: rgb(16, 16, 16);
color: rgb(255, 255, 255);

</string>
    </property>
    <property name="text">
     <string>INSERT NODE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>560</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
font: 12pt &quot;Arial&quot;;
background-color: rgb(16, 16, 16);
color: rgb(255, 255, 255);

</string>
    </property>
    <property name="text">
     <string>BOTH</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMessage">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Arial&quot;;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="lstInsertedNodes">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHorizontal">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
font: 12pt &quot;Arial&quot;;
background-color: rgb(16, 16, 16);
color: rgb(255, 255, 255);

</string>
    </property>
    <property name="text">
     <string>HORIZONTAL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVertical">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
font: 12pt &quot;Arial&quot;;
background-color: rgb(16, 16, 16);
color: rgb(255, 255, 255);

</string>
    </property>
    <property name="text">
     <string>VERTICAL</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
